lib/cli.rb

```diff
@@ -63,6 +63,16 @@ def to_ad(nepali_date)
       @shell.say(output_txt, @output_color)
     end
 
+    desc "next", "get remaining days for 1st of next month and last day of current month"
+    def next
```

Comment:
Quick question
The command "next" is unclear as to what the `miti` achieves. It's not a big deal, we do have documentation. I'm simply curious about the command. WDYT?

Reply:
Well, I couldn't think of better name. I agree that the name seems kind of unclear at first glance. My first thought was 'next1'. Can you think of something better?

Reply:
Well, I'm out of ideas too. Let's stick with this one for now, we do have documentation.

Reply:
Cool